### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by OR13**
- [x] add docker setup
- [x] add lint
- [x] add ci tests
- [x] add respec test result rendering
- [x] fix pr preview
- [x] add request fixture
- [x] add response fixture
- [x] automatically merge positive and negative tests

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-test-suite/pull/9.html" title="Last updated on Sep 27, 2020, 1:11 AM UTC (2d4502c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-test-suite/9/d21a3b5...2d4502c.html" title="Last updated on Sep 27, 2020, 1:11 AM UTC (2d4502c)">Diff</a>

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]
This is a fantastic body of work. I like the general structure of the test suite.
There will probably need to be more guidance for those wanting to test their DID Method implementations, as well as guidance for those wishing to contribute to the test suite itself, but neither of those things should hold up merging this PR imo.
I also had a couple of questions, but do not feel they need to be addressed before merging either.

File: packages/did-core-test-server/src/services/scenarios/index.js
Comment (OR13) [reply packages/did-core-test-server/src/services/scenarios/index.js:5]: yes, there should be. I had to stop somewhere for this PR, hoping this invites others to contribute.

File: index.html
Comment (OR13) [reply index.html:94]: initially i though assertions might "add up to" normative statements... it could be a MUST IMO.

**Review by msporny — APPROVED** [review]
Great start -- interested in seeing how easy it is to add tests and run Veres One against the test suite.

File: packages/did-core-test-server/src/services/scenarios/index.js
Comment (brentzundel) [reply packages/did-core-test-server/src/services/scenarios/index.js:5]: Cool. That's what I thought. 
And I totally agree that others should contribute.

File: index.html
Comment (brentzundel) [reply index.html:94]: That will be interesting to explore as the test suite begins to take form.

**Comment by OR13** [comment]
> Great start -- interested in seeing how easy it is to add tests and run Veres One against the test suite.

I suggest trying that sooner rather than later, it's easier to refactor while it's small.... I need to do the same for a bunch of other did methods, but in order for it to be useful we also need a few more tests i think.

**Comment by OR13** [comment]
@brentzundel @burnburn @msporny please help get this merged / make me person who can help move this along faster.

**Comment by msporny** [comment]
> @brentzundel @burnburn @msporny please help get this merged / make me person who can help move this along faster.

I don't have the authority to merge or edit settings for this repo. I expect that only @iherman and maybe the Chairs do.

Same request as @OR13 -- please allow me to merge things in this repo.

**Review by peacekeeper — APPROVED** [review]
+1 to merging, this is great!

I tried to contribute syntax tests here https://github.com/w3c/did-test-suite/tree/feat/add-syntax-scenario, but couldn't really figure out how to correctly design the scenarios and assertions to test positive/negative cases against an implementation.. Will probably need more help understanding it..

**Comment by OR13** [comment]
ping @iherman @brentzundel @burnburn

**Comment by brentzundel** [comment]
Multiple positive reviews, no objection raised. Merging.

**Pull request merged**
